src/dotnet/Chat.UI.Blazor/Services/AudioUI.cs

```diff
@@ -1,26 +1,102 @@
 using ActualChat.Audio;
 using ActualChat.Audio.UI.Blazor.Components;
+using ActualChat.UI.Blazor.Services;
 
 namespace ActualChat.Chat.UI.Blazor.Services;
 
-// TODO: extract into a separate class RecordingUI
-public partial class ChatUI
+public class AudioUI : WorkerBase
```

Comment:
Is it possible to move this AudioUI to Audio.UI.Blazor.Services without adding reference to Chats\Chat.UI?

Reply:
No, cause it depends on ActiveChatsUI